packages/moonstone/Slider/SliderBehaviorDecorator.js

```diff
@@ -0,0 +1,178 @@
+import {forward} from '@enact/core/handle';
+import hoc from '@enact/core/hoc';
+import Pause from '@enact/spotlight/Pause';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import $L from '../internal/$L';
+
+import {forwardSpotlightEvents} from './utils';
+
+const useHintOnActivate = ({active}) => {
+	return {
+		useHintText: active
+	};
+};
+
+const toggleActivate = ({active}) => {
+	return {
+		active: !active
+	};
+};
+
+const defaultConfig = {
+	// FIXME: This is a compromise to maintain a single decorator for Slider and IncrementSlider
+	// that handles both a consolidated focus state and spotlight directional event mgmt. When this
+	// is unset (for Slider), this decorator will listen to onKeyDown and fire spotlight events.
+	// When set (for IncrementSlider), it specifies the event that is passed down to trigger
+	// spotlight events and also doesn't remove the spotlight directional callbacks from the props
+	// so the Wrapped component can fire them manually or use the callback for the default behavior.
+	emitSpotlightEvents: null
+};
+
+// Adds moonstone-specific slider behaviors
```

Comment:
Yes it is. I didn't use JSdoc because the information is never intended to be consumed outside of the source. Consider it inline docs on the requirements for the component.